cli.js

```diff
@@ -170,6 +171,56 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async (argv) => {
     await handleCommand(argv, listHandler);
   })
+  .command('triggers', 'Manages triggers', yargs0 => yargs0
+      .demandCommand()
+      .command('upsert', 'Creates a new trigger or updates an existing one', (yargs1) => {
+        yargs1
+          .usage('Usage: $0 triggers upsert <trigger name> [options]')
+          .positional('trigger name', {
+            describe: 'Trigger name',
+            type: 'string',
+          })
+          .option('source-uri', {
+            demandOption: true,
+            describe: 'Redis-stream uri',
+            type: 'string',
+          })
+          .option('target-func', {
```

Comment:
why not simply "function"?

Reply:
There's also an error handler, I vote for calling it `target` and dropping the `func` part.

Reply:
changed to `target`